src/ui/components/results/universalresultscomponent.js

```diff
@@ -59,7 +59,7 @@ export default class UniversalResultsComponent extends Component {
       isPreSearch: searchState === SearchStates.PRE_SEARCH,
       isSearchLoading: searchState === SearchStates.SEARCH_LOADING,
       isSearchComplete: searchState === SearchStates.SEARCH_COMPLETE,
-      showNoResults: sections.length === 0 && query,
+      showNoResults: sections.length === 0 && (query || query === ''),
```

Comment:
Do we need to add `allowEmptySearch` in here at all? Like `(query || (query === ''" && allowEmptySearch)`. Or am I worried about an edge case that doesn't exist?

Reply:
The search component prevents empty searches from occurring if `allowEmptySearch `is false, so the extra logic in the universalresultscomponent isn't necessary